radicle-httpd/Cargo.toml

```diff
@@ -26,13 +26,13 @@ hyper = { version = "0.14.17", default-features = false }
 lexopt = { version = "0.2.1" }
 serde = { version = "1", features = ["derive"] }
 serde_json = { version = "1", features = ["preserve_order"] }
-siwe = { version = "0.5" }
 thiserror = { version = "1" }
 time = { version = "0.3.17" }
 tokio = { version = "1.21", default-features = false, features = ["macros", "rt-multi-thread"] }
 tower-http = { version = "0.3.4", default-features = false, features = ["trace", "cors", "set-header"] }
 tracing = { version = "0.1.37", default-features = false, features = ["std", "log"] }
 tracing-logfmt = { version = "0.2", optional = true }
+axum-auth = { version= "0.3.0", default-features = false, features = ["auth-bearer"] }
```

Comment:
These are alphabetically ordered

Reply:
This has been removed, since I want to focus in this PR on authorize the session.